samples/AspNetCore/GrpcServiceSample/GrpcServiceSample.csproj

```diff
@@ -5,8 +5,16 @@
     <RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
Can you rebase this PR or something? I'm seeing a bunch of commits/changes from things that have already been merged to master.

Reply:
done